src/main/java/com/passbook/sparkeighteen/config/SwaggerConfig.java

```diff
@@ -0,0 +1,38 @@
+package com.passbook.sparkeighteen.config;
+
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.RestTemplate;
+import springfox.documentation.builders.ApiInfoBuilder;
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.service.ApiInfo;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+@Configuration
+@EnableSwagger2
+public class SwaggerConfig {
+    @Bean
+    public Docket api() {
+        return new Docket(DocumentationType.SWAGGER_2)
+                .apiInfo(getApiInfo())
+                .select()
+                .apis(RequestHandlerSelectors.withClassAnnotation(RestController.class))
+                .paths(PathSelectors.any())
+                .build();
+    }
+
+    private ApiInfo getApiInfo() {
+
+        return new ApiInfoBuilder()
+                .title("Swagger API Doc")
+                .description("More description about the API")
```

Comment:
@Aashishverma06 
He is asking you to update the description line which make sense to end user who will be consuming your api.

Reply:
![PASSBOOK APPLICATION DOCUMENTATION](https://user-images.githubusercontent.com/100829219/173730728-0aef2b91-5fb7-4eb9-b402-3fe3321d103f.png)